src/status_im/contexts/onboarding/events.cljs

```diff
@@ -73,7 +74,8 @@
                                       :onboarding/navigated-to-enter-seed-phrase-from-screen
                                       :screen/onboarding.new-to-status)]]
                :dispatch-later [{:ms       constants/onboarding-generating-keys-animation-duration-ms
-                                 :dispatch [:init-root :screen/onboarding.enable-notifications]}]
+                                 :dispatch [:onboarding/navigate-to-enable-notifications
```

Comment:
do we have only one screen? or there are in onboarding flow after notifications?

Reply:
we do have the welcome screen

Reply:
then it should be changed as well

Reply:
@flexsurfer the changes have been updated now just waiting for your approval